datareservoirio/storage/storage.py

```diff
@@ -83,120 +74,88 @@ def put(self, df, target_url, commit_request):
         response.raise_for_status()
         return
 
-    def get(self, timeseries_id, start, end):
+    def get(self, target_url):
         """
-        Get a range of data for a timeseries.
+        Get a Pandas Dataframe from storage.
+
+        The REST API endpoint ``target_url`` returns a collection of blob (endpoint)
+        which are downloaded and merged in sequence.
 
         Parameters
         ----------
-        timeseries_id: guid
-            Unique id of the timeseries
-        start: long
-            Start time of the range, in nanoseconds since EPOCH
-        end: long
-            End time of the range, in nanoseconds since EPOCH
-
-        """
-        log.debug("getting day file inventory")
-        response = self._timeseries_api.download_days(timeseries_id, start, end)
-        df = self._downloader.get(response)
-        return df
-
-
-class BaseDownloader:
-    """
-    Series download strategy that will download Pandas DataFrame from provided
-    cloud backend.
-
-    Multiple chunks will be downloaded in parallel using ThreadPoolExecutor.
+        target_url : str
+            REST API URL as ``/data/days?start={start}&end={end}``
 
-    """
+        Returns
+        -------
+        df : pd.DataFrame
+            Pandas DataFrame with two columns, ``index`` and ``values``.
 
-    def __init__(self, cacheio=None):
-        self._cacheio = cacheio
+        """
+        response = self._session.request("GET", target_url)

```

Comment:
Any particular reason why you don't use the `get()` method? Just curious.

Reply:
No, not really. All the `.get`, `.post` etc methods are just thin wrappers around the `.request` method (which it self is a wrapper around `.send` method). 

That said, we should conform to using the high level methods such as `.post` etc instead of `request`, whenever possible.